### ISSUE TITLE
Joystick teleoperation

### ISSUE THREAD

**Issue by TSC21** [mavlink/mavros#133]
In bound. Will need some help in the RC override part.

**Comment by pmukherj** [comment]
Sorry! Moving offices over new years. 
I'm testing with all sorts of computers (odroids, TK1's etc.).  I'll take a crack at using a UDP bridge. 
I'm guessing I can use any of the telemetry connections into the pixhawk? In addition to the companion computer, I also have a wireless connection into one of the telemetry ports (as is standard with 3DR pixhawk).

**Comment by pmukherj** [comment]
Yup. Verified that the setpoints are actually working! The internal setpoints are accepting it, and the controller outputs are also visible. Now for flight tests :). Wish me luck.

**Comment by mhkabir** [comment]
@pmukherj You can connect the companion to your TELEM2 port and start it via extras.txt in your microSD card.
Good that the setpoints are working. Best of luck with the flights. 
Please check : http://www.pixhawk.org/dev/ros/mavros_offboard 
At the bottom there is blank code line, where you should add the mavros command you used to change to offboard mode ( as an option for people if they want to )

Also, can you please add the order for flight testing here : http://www.pixhawk.org/dev/offboard_control/testing

Just put in the points, and I'll clean it up.

**Comment by mhkabir** [comment]
@vooon The joystick plugin works fine, but crashes too badly if one of the axes aren't found or out of range, etc.

**Comment by vooon** [comment]
We can add min/max limitting.

Note that wan not a plugin, node.

**Comment by mhkabir** [comment]
True ;) 
I think I'll do it as a plugin, as the script/python seems rather confusing + unreliable.

**Comment by vooon** [comment]
I don't want plugin because it's reinventing the wheel. Maximum rewrite from python to C++.
But really not needed when mavteleop runs on desctop and controls OBC's mavros.

**Comment by TSC21** [comment]
@vooon have you thought in a way of solving [this](https://github.com/mavlink/mavros/blob/master/mavros_extras/scripts/mavteleop#L243)?
